test/FunctionalTests/Microsoft.OData.Core.Tests/ODataMessageWriterTests.cs

```diff
@@ -225,7 +222,7 @@ public void SupportsODataUtf8JsonWriter()
 
             IJsonWriterFactory factory = request.ServiceProvider.GetService<IJsonWriterFactory>();
             Assert.IsType<ODataUtf8JsonWriterFactory>(factory);
-            Assert.Equal("{\"@odata.context\":\"http://www.example.com/$metadata#Edm.String\",\"value\":\"This is a test \\u0438\\u044F\"}", output);
+            Assert.Equal("{\"@odata.context\":\"http://www.example.com/$metadata#Edm.String\",\"value\":\"This is a test ия\"}", output);

```

Comment:
Does this mean that `Default` was escaping `ия` while `RelaxedUnsafe` doesn't? What's the behaviour when using `JsonWriter`?

Reply:
Yes. Previously the default `ODataUtf8JsonWriter` uses `JavaScriptEncoder.Default` which escapes non-ASCII characters. The new default, `JavaScriptEncoder.UnsafeRelaxed...` does not.

Reply:
@habbes Let's keep an eye on this especially if the previous default writer was escaping such characters. Because it essentially changes what we write on the wire materially and one of the goals of ODL 8 was to try as much as possible to avoid that change the output payload. Luckily, someone can swap in the native `JsonWriter`.

Reply:
True. I'll have that explicitly mentioned in the release candidate blog post. Let's see if someone will bring that up.